Monika After Story/game/script-greetings.rpy

```diff
@@ -371,3 +371,55 @@ label monikaroom_greeting_post:
         set_keymaps()
     return
 
+label greeting_youarereal:
```

Comment:
You have some major indentation errors in this label. Here is a demonstration of how to do this correctly:
```
label this_is_label:
    python:
        try:
            <code>
        except: 
            <code>
    m "dialogue"
    if condition_is_true:
        menu:
            m "question"
            "choice":
                m "oh i see you choice"
            "other choice":
                m "i see u made other choice"
        m "dialogue after menu"
    else:
        m "dialogue in else"
    return
```
In addition, please make sure your editor is set to 4 spaces for indentation.